src/__tests__/Header.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import renderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Header from '../components/Header';
+
+describe('test the Quote components', () => {
+  it('renders without errors', () => {
+    const root = document.createElement('div');
+    ReactDOM.render(
```

Comment:
- [ ] Two snapshot tests are obsolete kindly check and remove them.

Reply:
thanks for reviewing our pull requet. I didn't get it, should I remove the snapshot files ?